### PR TITLE
Update xFormers MinGPT notebook reference

### PR DESCRIPTION
**Pull request by emmanuel-ferdman**
## What does this PR do?
Small PR - updates the xFormers MinGPT Colab notebook reference.

## Before submitting

- [x] Did you have fun?
  - Make sure you had fun coding 🙃
- [x] Did you read the [contributor guideline](https://github.com/facebookresearch/xformers/blob/master/CONTRIBUTING.md)?
- [ ] Was this discussed/approved via a Github issue? (no need for typos, doc improvements)
  - [ ] N/A
- [x] Did you make sure to update the docs?
  - [ ] N/A
- [ ] Did you write any new necessary tests?
  - [ ] N/A
- [ ] Did you update the [changelog](https://github.com/facebookresearch/xformers/blob/master/CHANGELOG.md)? (if needed)
  - [ ] N/A

## PR review
Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

**Comment by emmanuel-ferdman** [comment]
@danthe3rd, @lw, @zyan0 can you please review this PR?

File: docs/source/xformers_mingpt.ipynb [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "id": "view-in-github"
       },
       "source": [
-        "<a href=\"https://github.com/facebookresearch/xformers/blob/colab_example/docs/source/xformers_mingpt.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (lw) [inline docs/source/xformers_mingpt.ipynb:10]: I don't have much context on this, so I'm gonna ask some stupid questions. The old link was referring to a file in the `colab_example` branch, but that branch doesn't seem to exist anymore. The file in question, however, seems to exist in the `main` branch, under the same path. Is that the same file?

If so, why not just fix the name of the branch?

What do we gain from linking to the colab website?

Comment (emmanuel-ferdman) [reply docs/source/xformers_mingpt.ipynb:10]: You are correct, from the the branch name I assumed it should point to Colab, but I changed it to `main`.

**Review by lw — APPROVED** [review]
Great, thanks!

**Pull request merged**